SlackBotPrototype/SlackBotPrototype.sln

```diff
@@ -0,0 +1,40 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
Yes, from what I'm familiar with for the reason you hypothesized. I think if we wanted a definitive answer we could poke around Github and see the trends (`.sln` included or not)